sotodlib/preprocess/preprocess_util.py

```diff
@@ -911,6 +911,9 @@ def preproc_or_load_group(obs_id, configs_init, dets, configs_proc=None,
         Optional. Logger object or None will generate a new one.
     overwrite: bool
         Optional. Whether or not to overwrite existing entries in the preprocess manifest db.
+    save_archive :
+        Call cleanup_mandb if True to save to the archive and database files
+        in configs_init and configs_proc
```

Comment:
Add a statement here that says that this argument should not be used if this function is being called from within a multiproc or mpi script.